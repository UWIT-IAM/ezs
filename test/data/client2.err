mycert cn=localhost
server_reader
saving session
stdin: Success
using recovered session
mycert cn=localhost
server_reader
saving session
stdin: Success
