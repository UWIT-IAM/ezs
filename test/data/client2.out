Connection established to 'localhost:98766'.
> send 26 bytes
> send 15 bytes
> Connection established to 'localhost:98766'.
> send 27 bytes
> send 5 bytes
> 